src/components/Appointed.jsx

```diff
@@ -0,0 +1,247 @@
+        contact: 'AHAUBER@UCI.EDU',
+        linkedin: '...'
+    },
+    {
+        name: 'FirstName LastName',
+        position: 'Position',
+        photoUrl: GrayPlaceholder,
+        pronouns: 'THEY/THEM',
+        year: '? YEAR',
+        major: '???',
+        about: '...',
+        contact: '...',
+        linkedin: '...'
+    },
+    {
+        name: 'FirstName LastName',
+        position: 'Position',
+        photoUrl: GrayPlaceholder,
+        pronouns: 'THEY/THEM',
+        year: '? YEAR',
+        major: '???',
+        about: '...',
+        contact: '...',
+        linkedin: '...'
+    },
+    {
+        name: 'FirstName LastName',
+        position: 'Position',
+        photoUrl: GrayPlaceholder,
+        pronouns: 'THEY/THEM',
+        year: '? YEAR',
+        major: '???',
+        about: '...',
+        contact: '...',
+        linkedin: '...'
+    },
+    {
+        name: 'FirstName LastName',
+        position: 'Position',
+        photoUrl: GrayPlaceholder,
+        pronouns: 'THEY/THEM',
+        year: '? YEAR',
+        major: '???',
+        about: '...',
+        contact: '...',
+        linkedin: '...'
+    },
+    {
+        name: 'Gavin Bautista',
+        position: 'Cultural Director',
+        photoUrl: CulturalDirector,
+        pronouns: 'HE/HIM',
+        year: '2ND YEAR',
+        major: 'MECHANICAL ENGINEERING',
+        about: 'Hello! I’m Gavin Bautista and I am FUSION’s cultural director. I prioritize enhancing our organization\'s Cultural Pillar, guiding members in understanding cultural identity. I aim to foster a welcoming environment by sharing Filipino culture through workshops and resources, including history, traditions, languages, and more. Collaborating with the Alyansa Board, I seek to strengthen connections with sister organizations.',
+        contact: 'GCBAUTIS@UCI.EDU',
+        linkedin: '...'
+    },
+];
+
+const Appointed = () => {
+    const [selectedProfile, setSelectedProfile] = useState(null);
+
+    return (
+        <div className="bg-white text-black py-16">
+            <div className="text-center mb-8">
+                <div className='flex items-center justify-center mb-2'>
+                    <img src={HalfGear} alt="Half Gear" className="h-16 justify-center" />
+                </div>
+                <div className="flex items-center justify-center mb-2">
+                    <div className="border-t border-[#FAC80A] w-[115px] border-[2px] mr-2"></div>
+                    <h2 className="text-black text-[32px] font-karla font-bold uppercase leading-[37.41px] tracking-[0.12em]">Appointed</h2>
+                    <div className="border-t border-[#FAC80A] w-[115px] border-[2px] ml-2"></div>
+                </div>
+            </div>
+            <div className="grid grid-cols-1 sm:grid-cols-2 lg:grid-cols-4 gap-[20px] justify-items-center pb-[20px]">
```

Comment:
I believe the code is here, but looking at the appointed page it seems a little cramped,  looking at the figma the  groups of images are in the same width (comparing appointed and exec) so this would require the appointed pictures to be a little smaller so it can fit